src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,6 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
-    }
+public class ColorSupplier extends RandoM {
```

Comment:
Please pay attention to access modifiers. Classes, fields, constructors, methods must have access modifiers otherwise default will be used that isn't a good practice.